### ISSUE TITLE
Reading and writing FileGDB (filegdb driver) with Fiona (Geopandas) not working

### ISSUE THREAD

**Issue by Rbroekie** [Toblerity/Fiona#698]
Hello,

I am trying to use geopandas 0.4.0. to read and write with FileGDB. I already reported in the geopandas Github that I can't get it working, but no solution yet. I tried a lot of things already and searched a lot of blogs to find a solution but nothing helped yet. I see there are some people who get it working, so suppose it should be possible.  Because Fiona library is used to read and write and errors are fiona specific I wanted to try my luck here.

## Steps to reproduce the problem.

What I currently did is running these codes in the pyhon window of Arcgis to find out what is going wrong:
`
>>> c = fiona.open(r'C:\Users\broekaar\Documents.........gdb')
>>> c.driver
u'ESRI Shapefile'
>>> fiona.supported_drivers
{'PCIDSK': 'r', 'OpenFileGDB': 'r', 'SUA': 'r', 'GML': 'raw', 'ARCGEN': 'r', 'GeoJSON': 'rw', 'GPKG': 'rw', 'S57': 'r', 'Idrisi': 'r', 'GPX': 'raw', 'ESRI Shapefile': 'raw', 'SEGY': 'r', 'BNA': 'raw', 'CSV': 'raw', 'FileGDB': 'raw', 'AeronavFAA': 'r', 'GPSTrackMaker': 'raw', 'DGN': 'raw', 'MapInfo File': 'raw', 'DXF': 'raw'}
>>> fiona.open(r'C:\Users\broekaar\Documents\.............gdb', driver='FileGDB')
Runtime error 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python27\ArcGIS10.6\lib\site-packages\fiona\env.py", line 405, in wrapper
    return f(*args, **kwargs)
  File "C:\Python27\ArcGIS10.6\lib\site-packages\fiona\__init__.py", line 257, in open
    layer=layer, enabled_drivers=enabled_drivers, **kwargs)
  File "C:\Python27\ArcGIS10.6\lib\site-packages\fiona\collection.py", line 162, in __init__
    self.session.start(self, **kwargs)
  File "fiona\ogrext.pyx", line 467, in fiona.ogrext.Session.start
  File "fiona\_shim.pyx", line 73, in fiona._shim.gdal_open_vector
DriverError: C:\Users\broekaar\Documents\...............gdb: Permission denied
`

## Operating system

Windows 10.2018.3
64-bit Opereating system, x64-based processor
Python 2.7 32-bit

## Fiona and GDAL version and provenance

-the Fiona-1.8.3-cp27-cp27m-win32.whl installed from Gohlke using pip version 18.1.
-the GDAL-2.2.4-cp27-cp27m-win32 installed from Gohlke using pip version 18.1.
-geopandas 0.4.0
-Visual studios Community 2017
-the FileGDB_API_1_5_1-VS2017 downloaded. And FileGDBAPI.dll added to C:\Python27\ArcGIS10.6\Lib\site-packages\osgeo
- created GDAL_Driver_path for: C:\Python27\ArcGIS10.6\Lib\site-packages\osgeo\gdalplugins

If more info is need please let me know

**Comment by sgillies** [comment]
@Rbroekie as far as I can see, you've followed the complicated directions for setting up the plugin correctly. You're fortunate that Gohlke includes osgeo/gdalplugins/ogr_FileGDB.dll in the wheel because it is tricky to build plugins.

When I saw "permission denied" in the traceback (thanks for including it!) I did some googling and I think advice at https://community.esri.com/thread/17026 may apply. I'm going to close this issue as OGR plugins are out of scope for Fiona and because the project doesn't have any special solutions for them.

**Comment by Rbroekie** [comment]
Thanks @sgillies for the quick response. I know I followed the complicated direction but because I found some documentation on that I hoped to get it working. I also tried the 'easy' way via anaconda but got stuck in all kind of compiling issues too. In general it is a nightmare to get the FileGDB driver working. 

Considering your link to the Esri thread, I had a look at it but as far as I can see it is not really a general permission issue, at least it looks like I have all the permissions. And I can open the gdb with fiona (Esri_shapefile driver, which seems weird to me) but also the OpenFileGDB driver. Maybe a FileGDB driver permission issue? Well, I have no idea what the issue exactly is but because the OGR plugins are out of Fiona scope, I will try to see if I can find the solution elsewhere.
